src/main/java/com/minecolonies/util/StructureWrapper.java

```diff
@@ -592,6 +452,47 @@ public Block getBlock()
     }
 
     /**
+     * Creates the scan directories for the scanTool.
+     * @param world the worldIn.
+     */
+    public static void createScanDirectory(@NotNull World world)
+    {
+        File minecolonies;
+        if (world.isRemote)
+        {
+            minecolonies = new File(Minecraft.getMinecraft().mcDataDir, "minecolonies/");
+        }
+        else
+        {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'state' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)